.github/workflows/nomad.yaml

```diff
@@ -192,5 +171,4 @@ jobs:
         uses: actions/upload-artifact@v4
         with:
           name: summariser-report
-          if-no-files-found: ignore
```

Comment:
I assume this was only added to allow the summarizer to proceed.

Now a job is considered failed if no summary is generated.